### ISSUE TITLE
No restriction provided on community/collection logo

### ISSUE THREAD

**Issue by steph-ieffam** [DSpace/DSpace#9315]
**Describe the bug**
Test https://github.com/DSpace/DSpace/blob/main/dspace-server-webapp/src/test/java/org/dspace/app/rest/CommunityLogoControllerIT.java#L34-L45 & https://github.com/DSpace/DSpace/blob/main/dspace-server-webapp/src/test/java/org/dspace/app/rest/CollectionLogoControllerIT.java#L37-L50
make clear that there's no restriction when a Community/Collection logo is uploaded.
In these test cases TXT files are being set as logo for community/collection

My suggestion is to reuse https://github.com/DSpace/DSpace/blob/main/dspace-iiif/src/main/java/org/dspace/app/iiif/service/utils/IIIFUtils.java#L231-L236 to check is the Community/Collection logo is valid and throw and error when the format is invalid.

**Expected behavior**
Logo bitstream are restricted only to Images

**Comment by tdonohue** [comment]
Agreed, it'd be nice to restrict these to images.  Moving to the 7.6.x board .  Needs a volunteer
